sqllogictest-bin/src/engines/postgres.rs

```diff
@@ -59,6 +59,7 @@ impl sqllogictest::AsyncDB for Postgres {
                 || lower_sql.starts_with("show")
                 || lower_sql.starts_with("with")
                 || lower_sql.starts_with("describe")
+                || lower_sql.contains("returning")
```

Comment:
Constraint the command type according to https://www.postgresql.org/docs/current/dml-returning.html?
```suggestion
                || (lower_sql.starts_with("insert") || lower_sql.starts_with("update") || lower_sql.starts_with("delete")) && lower_sql.contains("returning")
```